frontend/src/lib/components/ProjectList.svelte

```diff
@@ -19,7 +19,7 @@
 				<p>{project.code}</p>
 
 				{#if project.userCount > 1}
-					<p>{t('projectlist.shared_with', project.userCount)}</p>
+					<p>{t('projectlist.shared_with', {memberCount: project.userCount})}</p>
```

Comment:
sorry you had to compromise on utilizing keys instead of the default content the way you wanted.

Reply:
actually I found an extension to help with that!